src/routes/pages/CartPage/CartPage.tsx

```diff
@@ -0,0 +1,83 @@
+import '../../../App.css';
+
+import Header from '../../../components/common/Header/Header';
+import HeaderButton from '../../../components/common/Header/HeaderButton';
+import ContainerLayout from '../../../components/common/ContainerLayout/ContainerLayout';
+import CartListTitle from '../../../components/CartListTitle/CartListTitle';
+import CartItem from '../../../components/CartItem/CartItem';
+import CartList from '../../../components/CartList/CartList';
+import CartPriceInfo from '../../../components/CartPriceInfo/CartPriceInfo';
+import OrderButton from '../../../components/OrderButton/OrderButton';
+import EmptyCart from '../../../components/EmptyCart/EmptyCart';
+
+import { Logo } from '../../../assets';
+import { cartPrice } from '../../../utils/cartPrice/cartPrice';
+
+import useCartList from '../../../hooks/useCartList';
+import useSelect from '../../../hooks/useSelect';
+import LoadingSpinner from '../../../components/common/LoadingSpinner/LoadingSpinning';
+
+function CartPage() {
+  const {
+    cartList,
+    isLoading,
+    increaseCartItem,
+    decreaseCartItem,
+    deleteCartItem,
+  } = useCartList();
+
+  const {
+    selectedItems,
+    isAllSelected,
+    handleSelectItem,
+    handleSelectAllItems,
+  } = useSelect(cartList);
+
+  const totalPrice = cartPrice.totalPrice(cartList, selectedItems);
+
+  const renderCartList = () => {
+    return cartList.length === 0 ? (
+      <EmptyCart />
+    ) : (
+      <>
+        <CartList
+          isAllSelected={isAllSelected}
+          handleSelectedAllItems={handleSelectAllItems}
+        >
+          {cartList.map((cartItem) => (
+            <CartItem
+              key={cartItem.id}
+              cartItem={cartItem}
+              isSelected={selectedItems.includes(cartItem.id)}
+              handleSelectItem={handleSelectItem}
+              increaseCartItem={increaseCartItem}
+              decreaseCartItem={decreaseCartItem}
+              deleteCartItem={deleteCartItem}
+            />
```

Comment:
전체적으로 상위 컴포넌트에서 정의한 이름을 하위 컴포넌트에 그대로 전달하고 있군요 ㅎㅎ
이렇게 하위 컴포넌트의 입장에서 props가 표현되면 어떨까요?


```suggestion
            <CartItem
              key={cartItem.id}
              {...cartItem}
              selected={selectedItems.includes(cartItem.id)}
              onSelect={handleSelectItem}
              onIncrementClick={increaseCartItem}
              onDecrementClick={decreaseCartItem}
              onDeleteClick={deleteCartItem}
            />
```